hibernate-core/src/main/java/org/hibernate/tool/schema/internal/SchemaCreatorImpl.java

```diff
@@ -122,19 +123,30 @@ public void release() {
 	}
 
 	@Override
-	public void doCreation(Metadata metadata, boolean createSchemas, Target... targets)
+	public void doCreation(Metadata metadata, boolean createNamespaces, Target... targets)
 			throws SchemaManagementException {
 		doCreation(
 				metadata,
-				createSchemas,
+				createNamespaces,
 				metadata.getDatabase().getJdbcEnvironment().getDialect(),
 				targets
 		);
 	}
 
 	@Override
-	public void doCreation(Metadata metadata, boolean createSchemas, Dialect dialect, Target... targets)
+	public void doCreation(Metadata metadata, boolean createNamespaces, Dialect dialect, Target... targets)
 			throws SchemaManagementException {
+		boolean tryToCreateCatalogs = false;
```

Comment:
I wonder if it makes sense to move the creation and dropping of namespaces into separate methods?